storage/tests/StorageManager_unittest.cpp

```diff
@@ -205,46 +205,4 @@ TEST(StorageManagerTest, DifferentNUMANodeBlobTestWithEviction) {
 }
 #endif  // QUICKSTEP_HAVE_LIBNUMA
 
-// Trigger an eviction from the same shard in StorageManager's
```

Comment:
@zuyu I will refactor it to test a similar situation, however the original situation will not happen.